src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/CyclomaticComplexityCheck.java

```diff
@@ -99,50 +99,166 @@
  * &lt;module name="CyclomaticComplexity"/&gt;
  * </pre>
  * <p>
- * To configure the check with a threshold of 15:
+ * Example:
+ * </p>
+ * <pre>
+ * class CyclomaticComplexity {
+ *   // Cyclomatic Complexity = 12
+ *   public void foo() { // 1, function declaration
+ *     if (a == b) { // 2, if
```

Comment:
please add necessary fields to make it compilable.
Also, there is no need to compare variables, it can be int variables compared with literals, like `a == 1`